student.rb

```diff
@@ -1,7 +1,9 @@
 require './person'
 
 class Student < Person
-  def initialize(age, classroom, name: 'Unknown', parent_permission: true)
+  attr_accessor :name, :classroom
+
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
   end
```

Comment:
- [ ] nice job. Please kindly add a method that serves as the 'belongs-to' side. that is, a student belongs to a classroom
```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

Reply:
@tufoinnkuo10 thank you for the review and comprehensive suggestions . I will implement the changes as soon as possible